src/AwsServiceProvider.php

```diff
@@ -26,7 +26,8 @@ class AwsServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        $source = realpath(__DIR__ . '/../config/aws.php');
+        $source = __DIR__.'/../config/aws.php';
```

Comment:
Maybe instead of `realpath` you could use `dirname` to walk up the directory tree a level? Something like:
```php
$source = implode(DIRECTORY_SEPARATOR, [dirname(__DIR__), 'config', 'aws.php']);
```